composer.json

```diff
@@ -21,7 +21,7 @@
         }
     },
     "require": {
-        "php": ">= 5.3",
```

Comment:
Is there a reason that this has been bumped up?

Reply:
PHP 5.* is not supported anymore https://www.php.net/supported-versions.php

Reply:
There are still people using it even tho its no longer supported and unless there is something in the library that requires a newer version I am trying to keep it compatible with older versions

Reply:
BTW I could see bumping this to 5.4 if needed as the dependent xmlseclibs library requires at least that version.